cypress/integration/images.spec.js

```diff
@@ -161,6 +172,86 @@ describe('Test all image insertion methods', () => {
 		})
 	})
 
+	it('test if image files are in the attachment folder', () => {
+		// check we stored the image names/ids
+		cy.expect(Object.keys(attachmentFileNameToId)).to.have.lengthOf(3)
+
+		cy.get(`#fileList tr[data-file="test.md"]`)
+			.should('have.attr', 'data-id')
+			.then((documentId) => {
+				cy.openFile('.attachments.' + documentId)
+				cy.wait(1000)
+				cy.screenshot()
+				for (const name in attachmentFileNameToId) {
+					cy.get(`#fileList tr[data-file="${name}"]`)
+						.should('exist')
+						.should('have.attr', 'data-id')
+						.should('eq', String(attachmentFileNameToId[name]))
+				}
+			})
+	})
+
+	it('test if attachment folder is moved with the markdown file', () => {
+		cy.createFolder('subFolder')
+		cy.wait(2000)
```

Comment:
There is a specific reason to use wait here?

Reply:
The reason was: to make sure the folder is created before going on. Tests were passing locally but failing in a slow context like the CI run.